flash/core/data/utilities/paths.py

```diff
@@ -98,16 +98,16 @@ def isdir(path: Any) -> bool:
         return False
 
 
-def list_subdirs(dir: PATH_TYPE) -> List[str]:
+def list_subdirs(folder: PATH_TYPE) -> List[str]:
```

Comment:
Mybe we should change the method to `list_subfolders` too?

Reply:
Will renaming to `list_subfolders` be a breaking change? If somewhere users are already using this helper function, we can keep 2 functions:

```python
def list_subdirs(folder: PATH_TYPE) -> List[str]:
    # Raise a DeprecationWarning here that this will be removed in a future release
    # and call list_subfolders(folder)

def list_subfolders(folder: PATH_TYPE) -> List[str]:
    # Implementation...
```

What do you think?

Reply:
let's make it another PR? :)